src/com/owncloud/android/lib/common/OwnCloudClient.java

```diff
@@ -54,135 +55,135 @@
-    
+
     private static int sIntanceCounter = 0;
     private boolean mFollowRedirects = true;
     private OwnCloudCredentials mCredentials = null;
     private int mInstanceNumber = 0;
-    
+
     private Uri mBaseUri;
 
     private OwnCloudVersion mVersion = null;
-    
+
     /**
      * Constructor
      */
     public OwnCloudClient(Uri baseUri, HttpConnectionManager connectionMgr) {
         super(connectionMgr);
-        
+
         if (baseUri == null) {
-        	throw new IllegalArgumentException("Parameter 'baseUri' cannot be NULL");
+            throw new IllegalArgumentException("Parameter 'baseUri' cannot be NULL");
         }
         mBaseUri = baseUri;
-        
+
         mInstanceNumber = sIntanceCounter++;
         Log_OC.d(TAG + " #" + mInstanceNumber, "Creating OwnCloudClient");
 
         String userAgent = OwnCloudClientManagerFactory.getUserAgent();
         getParams().setParameter(HttpMethodParams.USER_AGENT, userAgent);
         getParams().setParameter(
-        		PARAM_PROTOCOL_VERSION,
-        		HttpVersion.HTTP_1_1);
-        
-        getParams().setCookiePolicy(
-        		CookiePolicy.IGNORE_COOKIES);
+            PARAM_PROTOCOL_VERSION,
+            HttpVersion.HTTP_1_1
+        );
+
+        getParams().setCookiePolicy(CookiePolicy.IGNORE_COOKIES);
         getParams().setParameter(
-        		PARAM_SINGLE_COOKIE_HEADER, 			// to avoid problems with some web servers
-        		PARAM_SINGLE_COOKIE_HEADER_VALUE);
-        
+            PARAM_SINGLE_COOKIE_HEADER,             // to avoid problems with some web servers
+            PARAM_SINGLE_COOKIE_HEADER_VALUE
+        );
+
         applyProxySettings();
-        
+
         clearCredentials();
     }
 
-    
+
     private void applyProxySettings() {
-    	String proxyHost = System.getProperty("http.proxyHost");
-    	String proxyPortSt = System.getProperty("http.proxyPort");
-    	int proxyPort = 0;
-    	try {
-    		if (proxyPortSt != null && proxyPortSt.length() > 0) {
-    			proxyPort = Integer.parseInt(proxyPortSt);
-    		}
-    	} catch (Exception e) {
-    		// nothing to do here
-    	}
-
-    	if (proxyHost != null && proxyHost.length() > 0) {
-	    	HostConfiguration hostCfg = getHostConfiguration();
-	    	hostCfg.setProxy(proxyHost, proxyPort);
-	    	Log_OC.d(TAG, "Proxy settings: " + proxyHost + ":" + proxyPort);
-    	}
-	}
-
-
-	public void setCredentials(OwnCloudCredentials credentials) {
-    	if (credentials != null) {
-    		mCredentials = credentials;
-    		mCredentials.applyTo(this);
-    	} else {
-    		clearCredentials();
-    	}
+        String proxyHost = System.getProperty("http.proxyHost");
+        String proxyPortSt = System.getProperty("http.proxyPort");
+        int proxyPort = 0;
+        try {
+            if (proxyPortSt != null && proxyPortSt.length() > 0) {
+                proxyPort = Integer.parseInt(proxyPortSt);
+            }
+        } catch (Exception e) {
+            // nothing to do here
```

Comment:
It's a good practice to log the exceptions if are not thrown, as in this case. If the proxyPort String variable cannot be parsed to int, it will failed, and you are not handle the exception.